x/crosschain/module.go

```diff
@@ -175,8 +176,10 @@ func (AppModule) ConsensusVersion() uint64 { return 1 }
 
 // BeginBlock executes all ABCI BeginBlock logic respective to the capability module.
 func (am AppModule) BeginBlock(ctx sdk.Context, _ abci.RequestBeginBlock) {
-	if ctx.BlockHeader().Height == 946264 {
-		panic("pause at 946264; please revert back to v6.0.0 and restart")
+	if ctx.BlockHeight() == 946265 {
+		params := am.stakingKeeper.GetParams(ctx)
+		params.UnbondingTime = time.Hour * 24 * 4
```

Comment:
Something like an hour would also be safe if we want to keep the value low

Reply:
THe original plan was to have this as 4 days , I think the number we ended up with due to some sort of typo during the gov proposal